src/android/mod.rs

```diff
@@ -370,7 +370,7 @@ impl AndroidPackageReadout {
                     .iter()
                     .filter(|x| {
                         if let Some(ext) = extra::path_extension(x) {
-                            if ext == ".list" {
```

Comment:
@grtcdr I copied this from the `src/linux/mod.rs` file. Are your sure that the ext is ".list" and not "list" ?
In android while testing I found out that the ext is "list", at least for android.

Reply:
You are absolutely right, that's a mistake I wish I looked out for before publishing the new release.